Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/bbox.h

```diff
@@ -255,6 +272,66 @@ namespace CGAL {
       }
       return bb;
     }
+
+    /*!
+    * \ingroup PkgPolygonMeshProcessingRef
+    *
+    * adds an axis-aligned bounding box to a polygon mesh.
+    *
+    * @tparam PolygonMesh a model of `MutableFaceGraph`
+    * @tparam NamedParameters a sequence of \ref bgl_namedparameters "Named Parameters"
+    *
+    * @param pmesh a polygon mesh
+    * @param np an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
+    *
+    * \cgalNamedParamsBegin
+    *   \cgalParamNBegin{bbox_scaling}
+    *     \cgalParamDescription{a double used to scale the bounding box.
+    *       The default value is 1 and the bounding box is the smallest possible
+    *       axis-aligned bounding box.}
+    *     \cgalParamDefault{1.}
+    *     \cgalParamPrecondition{`bbox_scaling > 0`}
+    *   \cgalParamNEnd
+    *   \cgalParamNBegin{do_not_triangulate_faces}
+    *     \cgalParamDescription{a boolean used to specify if the bounding box faces
+    *       should be triangulated or not.
+    *       The default value is `true`, and faces are not triangulated.}
+    *     \cgalParamDefault{true}
+    *   \cgalParamNEnd
+    *   \cgalParamNBegin{vertex_point_map}
+    *     \cgalParamDescription{a property map associating points to the vertices of `pmesh`}
+    *     \cgalParamType{a class model of `ReadWritePropertyMap` with `boost::graph_traits<PolygonMesh>::%vertex_descriptor`
+    *                    as key type and `%Point_3` as value type}
+    *     \cgalParamDefault{`boost::get(CGAL::vertex_point, pmesh)`}
+    *     \cgalParamExtra{If this parameter is omitted, an internal property map for `CGAL::vertex_point_t`
+    *                     must be available in `PolygonMesh`.}
+    *   \cgalParamNEnd
+    *   \cgalParamNBegin{geom_traits}
+    *     \cgalParamDescription{an instance of a geometric traits class model of `Kernel`.}
+    *   \cgalParamNEnd
+    * \cgalNamedParamsEnd
+    *
+    * @see `bbox()`
+    */
+    template<typename PolygonMesh,
+             typename NamedParameters = parameters::Default_named_parameters>
+    void add_bbox(PolygonMesh& pmesh,
+                  const NamedParameters& np = parameters::default_values())
+    {
+      using parameters::choose_parameter;
+      using parameters::get_parameter;
+
+      using GT = typename GetGeomTraits<PolygonMesh, NamedParameters>::type;
+      using Iso_cuboid_3 = typename GT::Iso_cuboid_3;
+
+      PolygonMesh bbox_mesh;
+      CGAL::make_hexahedron(Iso_cuboid_3(bbox(pmesh, np)),
+                            bbox_mesh,
+                            np);
+      CGAL::copy_face_graph(bbox_mesh, pmesh,
+                            parameters::default_values(),
+                            np);
```

Comment:
You need Geom_traits::Construct_iso_cuboid_3.

Also can't that be reduced to:

```cpp
      CGAL::make_hexahedron(construct_isocuboid(PMP::bbox(pmesh, np), pmesh, np);
```
?